backend.tf

```diff
@@ -0,0 +1,99 @@
+terraform {
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "=2.51"
+    }
+  }
+    backend "azurerm" {
+        resource_group_name  = "OpenCohort1_IlinaGeorgieva_ProjectExercise"
+        storage_account_name = "illigeorgieva102"
+        container_name       = "illigeorgieva102"
+        key                  = "terraform.OpenCohort1_IlinaGeorgieva_ProjectExercise"
+    }
+
+}
+
+provider "azurerm" {
+  features {}
+}
+
+data "azurerm_resource_group" "main" {
+  name     = "OpenCohort1_IlinaGeorgieva_ProjectExercise"
+ 
+}
+
+resource "azurerm_app_service_plan" "main" {
+name = "terraformed-ilgdevops"
+location = data.azurerm_resource_group.main.location
+resource_group_name = data.azurerm_resource_group.main.name
+kind = "Linux"
+reserved = true
+sku {
+tier = "Basic"
+size = "B1"
+}
+}
+resource "azurerm_app_service" "main" {
+name = "illigeorgieva102"
+location = var.location
+resource_group_name = data.azurerm_resource_group.main.name
+app_service_plan_id = azurerm_app_service_plan.main.id
+
+site_config {
+app_command_line = ""
+linux_fx_version = "DOCKER|ilgeo/my-production-image:latest"
+}
+app_settings = {
+"DOCKER_REGISTRY_SERVER_URL" = "https://index.docker.io"
+"CONNECTIONSTRING" =var.CONNECTIONSTRING 
+"GITHUBID"=var.GITHUBID
+"WEBAPPLICATIONCLIENT" =var.WEBAPPLICATIONCLIENT
+"CLIENTSECRET" = var.CLIENTSECRET
+"TODOBOARD"= var.TODOBOARD
+"DOINGBOARD"=var.DOINGBOARD
+"DONEBOARD"=var.DONEBOARD
+
```

Comment:
Now that you've set up Loggly it would be nice to also set that here, e.g. add another variable for the `LOGGLY_TOKEN` and set that during your Travis script